providers/kind/kind.sh

```diff
@@ -36,7 +36,7 @@ declare -r KIND_REGISTRY_YAML="$KIND_DIR/local-registry.yml"
 declare -r KIND_KUBECONFIG="$KIND_DIR/kubeconfig"
 
 kind_preinstall_check() {
-	command -v kind || command -v kubectl || {
+	(command -v kind && command -v kubectl) || {
```

Comment:
@vprashar2929 this shouldn't require a subshell.  `( .. )` creates a new shell and executes the command. I know bash is weird at times :D